i18n/en.json

```diff
@@ -685,7 +685,8 @@
             "header": "$1 is currently blocked!",
             "noreason": "Blocked on $1 until $2 by $3.",
             "text": "Blocked on $1 until $2 by $3 with reason \"$4\".",
-            "until_infinity": "the end of all days"
+            "permabannednoreason": "Permanently blocked on $1 by $3."
+            "permabannedtext": "Permanently blocked on $1 by $3 with reason \"$4\".",
```

Comment:
Invalid JSON and not sorted correctly (alphabetical by key).

`until_infinity` is still in use for IP blocks.

Reply:
It's being replaced to make it correctly indicate a type of block if it's temp or not, but I've added a missing comma.